src/libxively/xi_debug.c

```diff
@@ -37,3 +38,29 @@ void xi_debug_data_logger_impl( const char* msg, const xi_data_desc_t* data_desc
 
     xi_free( tmp );
 }
+
+/** Trims the absolute file path to just the file name */
+const char* xi_debug_dont_print_the_path( const char* msg )
```

Comment:
That's interesting, I've never checked it - I don't know if this will work with majority of compilers - if we can provide such solution then I'm for it. The solution I've implemented does not depend on the compiler features... so we can explore compiler based solution.